src/Telemetry/AzureDevOpsClient/AzureDevOpsClient.cs

```diff
@@ -155,6 +162,40 @@ public async Task<WorkItem> CreateRcaWorkItem(string project, string title, Canc
             return JsonConvert.DeserializeObject<WorkItem>(json);
         }
 
+        public async Task<string> TryGetLogContents(string logUri, CancellationToken cancellationToken)
```

Comment:
These logs are huge. We should not return them as one giant string that needs to get allocated in memory.  This needs to be a Stream.

Reply:
Changed to using streams, had to do a bit of refactoring because of it